examples/component/demo/astro.config.mjs

```diff
@@ -2,6 +2,7 @@ import { defineConfig } from 'astro/config';
 
 // https://astro.build/config
 export default defineConfig({
-	// Comment out "renderers: []" to enable Astro's default component support.
-	renderers: [],
+	// Comment out "integrations: []" to disable
+	// Astro's default set of supported frameworks.
```

Comment:
Wait, isn't the default just... nothing now? I assumed `"integrations: []" and `integrations: undefined` were equivalent

Reply:
You're right! That's an old comment, I'll fix.